src/common/components/generics/Policy.tsx

```diff
@@ -0,0 +1,23 @@
+import { FormControl, Checkbox, HStack, Text } from '@chakra-ui/react';
+import { FormErrorMessage } from '@src/common/components';
+import { Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+export default function Policy({ name, control }) {
```

Comment:
So there are two different scenarios here.

if we are reusing this component else where, should it really be called `Policy`?

If we are not reusing it, then should it be a component under `@common` or does it belongs to its respective module?